src/lite-yt-embed.js

```diff
@@ -35,6 +38,13 @@ class LiteYTEmbed extends HTMLElement {
           this.style.backgroundImage = `url("${this.posterUrl}")`;
         }
 
+        if( this.title && this.title !== 'null' ) {
+            const titleEl = document.createElement( this.titleTag );
```

Comment:
also.. instead of adding a new DOM element. I think you can actually abuse the `lite-youtube::before` gradient pseudo element and set its `content` to the title string instead.

Reply:
Yup, awesome.